python313Packages.pysigma: 0.11.19 -> 0.11.23

fabaff opened the pull request:
Diff: https://github.com/SigmaHQ/pySigma/compare/refs/tags/v0.11.19...v0.11.20

Changelog: https://github.com/SigmaHQ/pySigma/releases/tag/v0.11.20

Diff: https://github.com/SigmaHQ/pySigma/compare/refs/tags/v0.11.20...refs/tags/v0.11.23

Changelog: https://github.com/SigmaHQ/pySigma/releases/tag/v0.11.23


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff committed:
python313Packages.pysigma: 0.11.19 -> 0.11.20

Diff: https://github.com/SigmaHQ/pySigma/compare/refs/tags/v0.11.19...v0.11.20

Changelog: https://github.com/SigmaHQ/pySigma/releases/tag/v0.11.20

petrzjunior commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 396771`

Logs: https://github.com/petrzjunior/nixpkgs-review-gha/actions/runs/14319270511


---
### `x86_64-linux`
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pysigma-backend-loki</li>
    <li>python312Packages.pysigma-backend-loki.dist</li>
    <li>python312Packages.pysigma-backend-sqlite</li>
    <li>python312Packages.pysigma-backend-sqlite.dist</li>
    <li>python313Packages.pysigma-backend-loki</li>
    <li>python313Packages.pysigma-backend-loki.dist</li>
    <li>python313Packages.pysigma-backend-sqlite</li>
    <li>python313Packages.pysigma-backend-sqlite.dist</li>
    <li>sigma-cli</li>
    <li>sigma-cli.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 36 packages built:</summary>
  <ul>
    <li>python312Packages.pysigma</li>
    <li>python312Packages.pysigma-backend-elasticsearch</li>
    <li>python312Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python312Packages.pysigma-backend-insightidr</li>
    <li>python312Packages.pysigma-backend-insightidr.dist</li>
    <li>python312Packages.pysigma-backend-opensearch</li>
    <li>python312Packages.pysigma-backend-opensearch.dist</li>
    <li>python312Packages.pysigma-backend-qradar</li>
    <li>python312Packages.pysigma-backend-qradar.dist</li>
    <li>python312Packages.pysigma-backend-splunk</li>
    <li>python312Packages.pysigma-backend-splunk.dist</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python312Packages.pysigma-pipeline-sysmon</li>
    <li>python312Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python312Packages.pysigma-pipeline-windows</li>
    <li>python312Packages.pysigma-pipeline-windows.dist</li>
    <li>python312Packages.pysigma.dist</li>
    <li>python313Packages.pysigma</li>
    <li>python313Packages.pysigma-backend-elasticsearch</li>
    <li>python313Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python313Packages.pysigma-backend-insightidr</li>
    <li>python313Packages.pysigma-backend-insightidr.dist</li>
    <li>python313Packages.pysigma-backend-opensearch</li>
    <li>python313Packages.pysigma-backend-opensearch.dist</li>
    <li>python313Packages.pysigma-backend-qradar</li>
    <li>python313Packages.pysigma-backend-qradar.dist</li>
    <li>python313Packages.pysigma-backend-splunk</li>
    <li>python313Packages.pysigma-backend-splunk.dist</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python313Packages.pysigma-pipeline-sysmon</li>
    <li>python313Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python313Packages.pysigma-pipeline-windows</li>
    <li>python313Packages.pysigma-pipeline-windows.dist</li>
    <li>python313Packages.pysigma.dist</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pysigma-backend-loki</li>
    <li>python312Packages.pysigma-backend-loki.dist</li>
    <li>python312Packages.pysigma-backend-sqlite</li>
    <li>python312Packages.pysigma-backend-sqlite.dist</li>
    <li>python313Packages.pysigma-backend-loki</li>
    <li>python313Packages.pysigma-backend-loki.dist</li>
    <li>python313Packages.pysigma-backend-sqlite</li>
    <li>python313Packages.pysigma-backend-sqlite.dist</li>
    <li>sigma-cli</li>
    <li>sigma-cli.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 36 packages built:</summary>
  <ul>
    <li>python312Packages.pysigma</li>
    <li>python312Packages.pysigma-backend-elasticsearch</li>
    <li>python312Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python312Packages.pysigma-backend-insightidr</li>
    <li>python312Packages.pysigma-backend-insightidr.dist</li>
    <li>python312Packages.pysigma-backend-opensearch</li>
    <li>python312Packages.pysigma-backend-opensearch.dist</li>
    <li>python312Packages.pysigma-backend-qradar</li>
    <li>python312Packages.pysigma-backend-qradar.dist</li>
    <li>python312Packages.pysigma-backend-splunk</li>
    <li>python312Packages.pysigma-backend-splunk.dist</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python312Packages.pysigma-pipeline-sysmon</li>
    <li>python312Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python312Packages.pysigma-pipeline-windows</li>
    <li>python312Packages.pysigma-pipeline-windows.dist</li>
    <li>python312Packages.pysigma.dist</li>
    <li>python313Packages.pysigma</li>
    <li>python313Packages.pysigma-backend-elasticsearch</li>
    <li>python313Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python313Packages.pysigma-backend-insightidr</li>
    <li>python313Packages.pysigma-backend-insightidr.dist</li>
    <li>python313Packages.pysigma-backend-opensearch</li>
    <li>python313Packages.pysigma-backend-opensearch.dist</li>
    <li>python313Packages.pysigma-backend-qradar</li>
    <li>python313Packages.pysigma-backend-qradar.dist</li>
    <li>python313Packages.pysigma-backend-splunk</li>
    <li>python313Packages.pysigma-backend-splunk.dist</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python313Packages.pysigma-pipeline-sysmon</li>
    <li>python313Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python313Packages.pysigma-pipeline-windows</li>
    <li>python313Packages.pysigma-pipeline-windows.dist</li>
    <li>python313Packages.pysigma.dist</li>
  </ul>
</details>


---
### `x86_64-darwin`
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pysigma-backend-loki</li>
    <li>python312Packages.pysigma-backend-loki.dist</li>
    <li>python312Packages.pysigma-backend-sqlite</li>
    <li>python312Packages.pysigma-backend-sqlite.dist</li>
    <li>python313Packages.pysigma-backend-loki</li>
    <li>python313Packages.pysigma-backend-loki.dist</li>
    <li>python313Packages.pysigma-backend-sqlite</li>
    <li>python313Packages.pysigma-backend-sqlite.dist</li>
    <li>sigma-cli</li>
    <li>sigma-cli.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 36 packages built:</summary>
  <ul>
    <li>python312Packages.pysigma</li>
    <li>python312Packages.pysigma-backend-elasticsearch</li>
    <li>python312Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python312Packages.pysigma-backend-insightidr</li>
    <li>python312Packages.pysigma-backend-insightidr.dist</li>
    <li>python312Packages.pysigma-backend-opensearch</li>
    <li>python312Packages.pysigma-backend-opensearch.dist</li>
    <li>python312Packages.pysigma-backend-qradar</li>
    <li>python312Packages.pysigma-backend-qradar.dist</li>
    <li>python312Packages.pysigma-backend-splunk</li>
    <li>python312Packages.pysigma-backend-splunk.dist</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python312Packages.pysigma-pipeline-sysmon</li>
    <li>python312Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python312Packages.pysigma-pipeline-windows</li>
    <li>python312Packages.pysigma-pipeline-windows.dist</li>
    <li>python312Packages.pysigma.dist</li>
    <li>python313Packages.pysigma</li>
    <li>python313Packages.pysigma-backend-elasticsearch</li>
    <li>python313Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python313Packages.pysigma-backend-insightidr</li>
    <li>python313Packages.pysigma-backend-insightidr.dist</li>
    <li>python313Packages.pysigma-backend-opensearch</li>
    <li>python313Packages.pysigma-backend-opensearch.dist</li>
    <li>python313Packages.pysigma-backend-qradar</li>
    <li>python313Packages.pysigma-backend-qradar.dist</li>
    <li>python313Packages.pysigma-backend-splunk</li>
    <li>python313Packages.pysigma-backend-splunk.dist</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python313Packages.pysigma-pipeline-sysmon</li>
    <li>python313Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python313Packages.pysigma-pipeline-windows</li>
    <li>python313Packages.pysigma-pipeline-windows.dist</li>
    <li>python313Packages.pysigma.dist</li>
  </ul>
</details>


---
### `aarch64-darwin`
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pysigma-backend-loki</li>
    <li>python312Packages.pysigma-backend-loki.dist</li>
    <li>python312Packages.pysigma-backend-sqlite</li>
    <li>python312Packages.pysigma-backend-sqlite.dist</li>
    <li>python313Packages.pysigma-backend-loki</li>
    <li>python313Packages.pysigma-backend-loki.dist</li>
    <li>python313Packages.pysigma-backend-sqlite</li>
    <li>python313Packages.pysigma-backend-sqlite.dist</li>
    <li>sigma-cli</li>
    <li>sigma-cli.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 36 packages built:</summary>
  <ul>
    <li>python312Packages.pysigma</li>
    <li>python312Packages.pysigma-backend-elasticsearch</li>
    <li>python312Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python312Packages.pysigma-backend-insightidr</li>
    <li>python312Packages.pysigma-backend-insightidr.dist</li>
    <li>python312Packages.pysigma-backend-opensearch</li>
    <li>python312Packages.pysigma-backend-opensearch.dist</li>
    <li>python312Packages.pysigma-backend-qradar</li>
    <li>python312Packages.pysigma-backend-qradar.dist</li>
    <li>python312Packages.pysigma-backend-splunk</li>
    <li>python312Packages.pysigma-backend-splunk.dist</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python312Packages.pysigma-pipeline-sysmon</li>
    <li>python312Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python312Packages.pysigma-pipeline-windows</li>
    <li>python312Packages.pysigma-pipeline-windows.dist</li>
    <li>python312Packages.pysigma.dist</li>
    <li>python313Packages.pysigma</li>
    <li>python313Packages.pysigma-backend-elasticsearch</li>
    <li>python313Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python313Packages.pysigma-backend-insightidr</li>
    <li>python313Packages.pysigma-backend-insightidr.dist</li>
    <li>python313Packages.pysigma-backend-opensearch</li>
    <li>python313Packages.pysigma-backend-opensearch.dist</li>
    <li>python313Packages.pysigma-backend-qradar</li>
    <li>python313Packages.pysigma-backend-qradar.dist</li>
    <li>python313Packages.pysigma-backend-splunk</li>
    <li>python313Packages.pysigma-backend-splunk.dist</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python313Packages.pysigma-pipeline-sysmon</li>
    <li>python313Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python313Packages.pysigma-pipeline-windows</li>
    <li>python313Packages.pysigma-pipeline-windows.dist</li>
    <li>python313Packages.pysigma.dist</li>
  </ul>
</details>

fabaff committed:
python313Packages.pysigma: 0.11.20 -> 0.11.23

Diff: https://github.com/SigmaHQ/pySigma/compare/refs/tags/v0.11.20...refs/tags/v0.11.23

Changelog: https://github.com/SigmaHQ/pySigma/releases/tag/v0.11.23

fabaff committed:
python312Packages.pysigma-backend-sqlite: refactor

fabaff committed:
python312Packages.pysigma-backend-sqlite: 0.2.0 -> 0.2.0-unstable-2025-01-21

fabaff committed:
python313Packages.pysigma-backend-loki: disable out-dated tests

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 396771`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 46 packages built:</summary>
  <ul>
    <li>python312Packages.pysigma</li>
    <li>python312Packages.pysigma-backend-elasticsearch</li>
    <li>python312Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python312Packages.pysigma-backend-insightidr</li>
    <li>python312Packages.pysigma-backend-insightidr.dist</li>
    <li>python312Packages.pysigma-backend-loki</li>
    <li>python312Packages.pysigma-backend-loki.dist</li>
    <li>python312Packages.pysigma-backend-opensearch</li>
    <li>python312Packages.pysigma-backend-opensearch.dist</li>
    <li>python312Packages.pysigma-backend-qradar</li>
    <li>python312Packages.pysigma-backend-qradar.dist</li>
    <li>python312Packages.pysigma-backend-splunk</li>
    <li>python312Packages.pysigma-backend-splunk.dist</li>
    <li>python312Packages.pysigma-backend-sqlite</li>
    <li>python312Packages.pysigma-backend-sqlite.dist</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike</li>
    <li>python312Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python312Packages.pysigma-pipeline-sysmon</li>
    <li>python312Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python312Packages.pysigma-pipeline-windows</li>
    <li>python312Packages.pysigma-pipeline-windows.dist</li>
    <li>python312Packages.pysigma.dist</li>
    <li>python313Packages.pysigma</li>
    <li>python313Packages.pysigma-backend-elasticsearch</li>
    <li>python313Packages.pysigma-backend-elasticsearch.dist</li>
    <li>python313Packages.pysigma-backend-insightidr</li>
    <li>python313Packages.pysigma-backend-insightidr.dist</li>
    <li>python313Packages.pysigma-backend-loki</li>
    <li>python313Packages.pysigma-backend-loki.dist</li>
    <li>python313Packages.pysigma-backend-opensearch</li>
    <li>python313Packages.pysigma-backend-opensearch.dist</li>
    <li>python313Packages.pysigma-backend-qradar</li>
    <li>python313Packages.pysigma-backend-qradar.dist</li>
    <li>python313Packages.pysigma-backend-splunk</li>
    <li>python313Packages.pysigma-backend-splunk.dist</li>
    <li>python313Packages.pysigma-backend-sqlite</li>
    <li>python313Packages.pysigma-backend-sqlite.dist</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike</li>
    <li>python313Packages.pysigma-pipeline-crowdstrike.dist</li>
    <li>python313Packages.pysigma-pipeline-sysmon</li>
    <li>python313Packages.pysigma-pipeline-sysmon.dist</li>
    <li>python313Packages.pysigma-pipeline-windows</li>
    <li>python313Packages.pysigma-pipeline-windows.dist</li>
    <li>python313Packages.pysigma.dist</li>
    <li>sigma-cli</li>
    <li>sigma-cli.dist</li>
  </ul>
</details>
